registry/api/errcode/errors.go

```diff
@@ -160,9 +187,7 @@ func (errs Errors) Len() int {
 	return len(errs)
 }
 
-// jsonError extends Error with 'Message' so that we can include the
-// error text, just in case the receiver of the JSON doesn't have this
-// particular ErrorCode registered
+// jsonError wrappers Errors with the json 'errors' name
 type jsonError struct {
```

Comment:
Can't this just be replaced by `Error` in the anonymous struct in the `Unmarshal`?